src/main_cfe_aorc_pet_rzAET.cxx

```diff
@@ -0,0 +1,293 @@
+    3. Update the PET model
+    3. Getting forcing from AORC and setting forcing for CFE
+    4. Getting PET from PET and setting for CFE
+    5. Get ice fraction from freeze-thaw model
+    5. Update the CFE model.
+    6. Update BMI Coupler to get updated soil storage/change for the SFT model
+    7. Update the Freeze-thaw model (soil temperature/ice content update)
+  ************************************************************************/
+
+  int n_soil_layers = cfe->soil_reservoir.n_soil_layers;
+
+  /************************************************************************
+    Now loop through time and call the models with the intermediate get/set
+  ************************************************************************/
+  printf("looping through and calling updata\n");
+  if (cfe->verbosity > 0)
+    print_cfe_flux_header();
+  for (int i = 0; i < 24862; i++){
+    
+    aorc_bmi_model->update(aorc_bmi_model);                         // Update model 1
+
+    pass_forcing_from_aorc_to_pet(pet_bmi_model, aorc_bmi_model);   // Get and Set values
+
+    pet_bmi_model->update(pet_bmi_model);
+
+    pass_forcing_from_aorc_to_cfe(cfe_bmi_model, aorc_bmi_model);   // Get and Set values
+
+    pass_pet_to_cfe(cfe_bmi_model, pet_bmi_model);   // Get and Set values
+
+    pass_smc_from_coupler_to_cfe(cfe_bmi_model,coupler_bmi,n_soil_layers);
+
+    if (pet->aorc.air_temperature_2m_K != aorc->aorc.air_temperature_2m_K){
+      printf("ERROR: Temperature values do not match from AORC and PET\n");
+      printf("Temperature value from AORC is %lf\n", aorc->aorc.air_temperature_2m_K);
+      printf("Tempterature value from PET is %lf\n", pet->aorc.air_temperature_2m_K);
+    }
+    
+    if (cfe->aorc.precip_kg_per_m2 != aorc->aorc.precip_kg_per_m2){
+      printf("Precip values do not match\n");
+      printf("precip value from AORC is %lf\n", aorc->aorc.precip_kg_per_m2);
+      printf("precip value from CFE is %lf\n", cfe->aorc.precip_kg_per_m2);
+    }
+    
+    if (cfe->et_struct.potential_et_m_per_s != pet->pet_m_per_s){
+      printf("ERROR: PET values do not match from PET and CFE\n");
+      printf("PET value from PET is %8.9lf\n", pet->pet_m_per_s);
+      printf("PET value from CFE is %8.9lf\n", cfe->et_struct.potential_et_m_per_s);
+    }
+
+    if (cfe->verbosity > 2){
+      printf("PET value from PET is %8.9lf\n", pet->pet_m_per_s);
+      printf("PET value from CFE is %8.9lf\n", cfe->et_struct.potential_et_m_per_s);
+    }
+    
+    cfe_bmi_model->update(cfe_bmi_model);                           // Update model 2
+  
+    if (cfe->verbosity > 0)
+      print_cfe_flux_at_timestep(cfe);
+    
+
+  }
+
+  // Run the Mass Balance check
+  mass_balance_check(cfe);
+  /************************************************************************
+    Finalize both the CFE and AORC bmi models
+  ************************************************************************/
+  printf("Finalizing BFE and AORC models\n");
```

Comment:
"CFE" instead of "BFE"

Reply:
this file has been deleted (not needed).